src/server/routes/v2/featuredPackages.ts

```diff
@@ -0,0 +1,83 @@
+import { FastifyInstance } from "fastify";
+import { PackageService } from "../../../database";
+
+export default async (fastify: FastifyInstance): Promise<void> => {
+  const packageService = new PackageService();
+
+  const updateFeaturedSchema = {
+    body: {
+      type: "object",
+      required: ["id", "banner", "logo"],
+      properties: {
+        id: {
+          type: "string",
+          minLength: 1,
+        },
+        banner: {
+          type: "string",
+          minLength: 1,
+        },
+        logo: {
+          type: "string",
+          minLength: 1,
+        },
+      },
+    },
+  };
+
+  const deleteFeaturedSchema = {
+    querystring: {
+      type: "object",
+      required: ["id"],
+      properties: {
+        id: {
+          type: "string",
+          minLength: 1,
+        },
+      },
+    },
+  };
+
+  //* get featured packages
+  fastify.get("/", async () => {
+    const featuredPackages = await packageService.find({
+      filters: { Featured: true },
+    });
+
+    return featuredPackages;
+  });
+
+  //* update featured package
+  fastify.post("/", { schema: updateFeaturedSchema }, async (req, res) => {
+    const { id, banner, logo } = req.body;
+    const pkg = await packageService.findOne({ filters: { Id: id } });
+
+    if (pkg != null) {
+      pkg.Featured = true;
+      pkg.Banner = banner;
+      pkg.Logo = logo;
+
+      await packageService.upsertPackage(pkg);
+      return res.code(200).send("updated featured package details");
+    }
+
+    return res.code(500).send("failed to update featured package details");
+  });
+
+  //* delete featured packaged (clear image, set to false)
+  fastify.delete("/", { schema: deleteFeaturedSchema }, async (req, res) => {
+    const { id } = req.query;
+    const pkg = await packageService.findOne({ filters: { Id: id } });
+
+    if (pkg != null) {
+      pkg.Featured = false;
+      pkg.Banner = "";
+      pkg.Logo = "";
+
+      await packageService.upsertPackage(pkg);
+      return res.code(200).send("deleted featured package details");
+    }
+
+    return res.code(500).send("failed to delete featured package details");
```

Comment:
another thing about returning data/codes, if youre returning data, a return at the end is fine (no need for send), if there is no return at the end, if the execution hits the end of the function, it *should* return a 200 with no body afaik